build/dt-report-generator-bundle.js

```diff
@@ -7,13 +7,38 @@
 
 const browserify = require('browserify');
 const fs = require('fs');
+const path = require('path');
+const rimraf = require('rimraf');
 
-const distDir = __dirname + '/../dist';
-const outFile = `${distDir}/report-generator.js`;
+const distDir = path.join(__dirname, '..', 'dist', 'dt-resources');
```

Comment:
can we rename these so they match the other build scripts (and the build script name goes with the output directory name).

what about
this script -> `build-dt-report-resources.js`
distDir -> `dist/dt-report-resources/`

?

Reply:
done